src/test/resources/suites/calculator.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<suite name="calc" verbose="1" parallel="true" thread-count="3">
+   <test name="Regression suite 1">
+      <groups>
+         <run>
+            <!-- <include name="basic"/> -->
```

Comment:
In hometask they asked to have an xml with 'include' element, so i added it but commented because it does not run all scenarios, just a demonstration that i can :)